### ISSUE TITLE
Admin. efficiency, UN peacekeeping, and national focuses

### ISSUE THREAD

**Issue by Aguis34** [simsulla/nwomod#1190]
Hi, I'm new to both the platform and the New World Order mod so forgive me if I'm doing anything wrong. I have a couple of issues, I'll start with the simplest one.

1. UN Peacekeeping missions. I like the way they're implemented but there's a minor problem, and it's that if there are two requests for UN aid within two years of one another, and one fails, so will the other. For instance, playing as Zaire I was asked to help South Sudan, and readily accepted. A few months later Gambia requested a UN mission, and knowing that they're a one province minor I refused, since the rebels would win before I get there. Obviously the mission failed, but I got the event (loss of prestige, etc.) as if I had accepted. I'm pretty sure this happens because there is only one UN peacekeeping flag, and it's universal, so if one mission fails, so will all others. The obvious solution would be to have unique flags for each country (e.g. UN_peacekeeping_gambia), but I don't know how hard it would be to implement. Another solution could be having numbers for the mission flags; for instance, if there's no current mission, the flag would be UN_peacekeeping_1 or something like that, but if that flag is already in place then you would have UN_peacekeeping_2 instead, and so on. They would always be unique this way, and will fail independently.

2. Money. I've played as three countries so far, Zaire (1992), Burma (1992), and Yugoslavia (1946). With all three I was losing ~300 pounds a day, with all sliders to the minimum, and maximum taxes. There was basically no income, and I had to cheat for money for the first 5 years or so until my finances stabilized. After that I could manage to keep in the positive (and get extremely rich sometimes), but the initial, inevitable bankruptcy is still anoying unless you cheat. I think the reason for this is my third issue.

3. Administrative efficiency. It is *extremely* low in all three starts. I know Yugoslavia, Zaire and Burma are not particularly rich countries but they should be playable at least. I have yet to check if this is true with major powers, but admin efficiency was usually around 6%. This could be the reason for the ridiculous loss of money early game, and I believe it stems from the fact that bureaucrats were 0.1% of the population, or even less sometimes. I tried to use national focus to encourage bureaucrats, which leads me to issue 4.

4. National focuses. I can't be 100% sure, but they look broken. Most of them won't do anything. Promoting bureaucrats with maxed admin. spending for 20 years would increase its number from 0.1% to 0.2% or, sometimes, 0.3%. Which looks more like the maxed admin. spending doing its thing and not really the effect of the national focus. The covert action to increase party loyalty in foregin countries is also not working for me. This one I'm sure about because after years of promoting liberalism in Chad the party loyalty map mode shows a "neutral" loyalty to the Liberal party, the same as every other. Maybe it is the map mode that doesn't work, but it does show me the real loyalty in other countries (in New York for instance), so it looks like it is the NF that isn't working.

I don't mean to undermine your work or anything, quite the opposite, I'm just trying to help its development by offering (what I believe to be) some constructive criticism or informing of problems I notice. I love the mod, it is a great idea and is very well implemented. It gives the game much more "replayability" while we wait for Victoria 3, and it's an amazing game on its own too, truly one of only a handful of games set in the current era. So thank you for that :)

**Comment by Aguis34** [comment]
Also, I forgot one about nukes. The AI seems too willing to use them (or I just happen to be very unlucky). In two of my three games I got nukes within 10 years of gameplay. As Burma it was kind of understandable, since I was at war with the USA (because I was allied to China and she tried to reclaim Taiwan), although I don't think minors should be the ones being nuked, just China (or other military powerhouses on their side). But as Zaire it was during the Rwanda war. It was really stupid, France joined them and not two months later she nuked three provinces of mine. I mean, that's ridiculous. The USA didn't nuke anyone since WWII, and they have gone to much more cruent and/or important wars than a conflict between Rwanda and Zaire. I mean, maybe there should be very strict requirements for using nuclear bombs (like having a % of your territory occupied? Thus Vietnam could war its civil war without the US nuking the hell out of them, because Vietnam doesn't need to occupy mainland America).

**Comment by arandomperson5000** [comment]
I completely agree with all of your points
1. For the UN peacekeeping problem, I think your suggestion of the country flags being numbered in those events is the most practical solution to this problem as adding the name of every single country to the country flags would be too time consuming.
2. and 3. I've been trying for a while now to solve this problem by adding a fixed amount of pounds to the files of each country based on the historical gdp either in 1946 or 1992 in US dollars however this method didn't work. Your suggestion of that administrative efficiency is causing this problem is quite interesting, I will have a look into it later. although for Zaire, I suppose that its current economic status in the 1992 bookmark of NWO is quite accurate as from what I know the Zairan government led by Mobutu was extremely corrupt with the country's economy imploding in the 1980s causing the average income of the average Zairan to be much lower than they would have had several decades ago.
4. I wasn't really aware of this issue, I will take a look into it.

As for the nuclear weapons system,  I'm planning to add an updated Nuclear weapon system in a future version where the AI would be less willing to use nuclear weapons on due to Mutually Assured Destruction however as of right now I could edit the current system where the event that allows countries to use nuclear weapons will have countries that aren't secondary or great powers will be exempt for being hit by nuclear weapons.

**Comment by Aguis34** [comment]
Thanks for the reply. By the way, are you a developer or something? I'm asking because your username is counterintuitive in that regard, haha. Anyway, you are right about Zaire being poor, but I'm not sure about Yugoslavia. That's why I also tried 1992 Netherlands and 1946 Canada.

As the Netherlands I couldn't stop losing money no matter what I did, although it was about 50 or 80 pounds a day, not 300. As Canada, I could start making money, but only after having *every* slider set to the minimum, all taxes maxed, and at least 50% tariffs. Admin efficiency as Netherlands was 11.6%, very low but still double that of Zaire or Yugoslavia, and 36% as Canada, which might be why it was the only country that could make money (although really, having education, admin. spending, and social spending on 0% while taxes and tariffs are maxed is not really a good long-term solution). I tried those two countries instead of Soviet Union, USA, or China because that way the gigantic industrial prowess of those countries doesn't create a false equivalent to my previous games.

Regarding nukes, I think your solution could work, but you should probably add the possibility to nuke countries that have the ability to do so themselves. Otherwise Israel (a minor nation) might be nuking Iran (a secondary power) without Iran being able to fire back.

**Comment by arandomperson5000** [comment]
Yes I'm a developer for NWO, for Yugoslavia in the 1946 bookmark, you're right about it as even though it was completely devastated during WW2, it would have a much less corrupt administration, better infrastructure, larger industries and a better economy than Zaire.

Also which version of NWO are you playing on as in the previous version, there was a major overhaul of the economy which divided artisians up into different services like working in transportation etc. However recently it was discovered that it had kinda messed up the economy so it could be a cause of that problem too.

As for Nukes I agree with you that there should be add the possibility to nuke countries that have the ability to do so themselves

**Comment by Aguis34** [comment]
I'm playing on 1.13 (the last one, I believe. I downloaded it like a month ago or so anyway). I didn't know about that overhaul you mention, it might have something to do with this. I can't really know, as I said before I'm new to the mod and have only played the latest version (apart from one game like two years ago which I didn't really enjoy at the time).

**Comment by arandomperson5000** [comment]
I apologise for the lack of work on this issue as I'm personally quite busy with work and exams currently right now, although I may have some free time to incorporate these future changes into 2.00
